### PR TITLE
chore: Set bundled NODE_ENV default to production.

### PR DESCRIPTION
**Pull request by KvelaGorrrrnio**
When ClipTool is run from a packaged executable, the NODE_ENV should be set to `production` if not already specified through environment variables.

### CODE REVIEW

**Comment by andr9528** [comment]
I have packaged and testet locally that changes in this PR appears to be working as intended.

I ran Cliptool both as a standalone .exe and through CasperCG Launcher.
In both cases the logs i saw was using the (json) structure applied while the NODE_ENV is 'production'.

**Review by andr9528 — APPROVED** [review]
Looking forward to getting these changes in. The error related to capital start letter of 'hiddenFileStorage.ts' is annoying me each time i switch branch...

File: package.json [hunk 1]
```diff
@@ -27,9 +27,9 @@
         "build-server": "tsc -p ./src/server",
         "build-gateway": "tsc -p ./src/gateway",
         "build-client": "cross-env NODE_ENV=production TS_NODE_PROJECT=\"./src/client/tsconfig.json\" webpack && copyfiles -f ./src/client/index.html ./build/client/",
-        "start": "node ./build/server/index.js",
-        "start-dev": "node --inspect ./build/server/index.js",
-        "start-local": "cross-env LOG_LEVEL=trace node --inspect ./build/server/index.js",
+        "start": "cross-env NODE_ENV=production node ./build/server/index.js",
```
Comment (LindvedKrvang) [inline package.json:30]: So if I run a `yarn start` on my local machine, I would run it as production?

**Review by LindvedKrvang — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (andr9528) [reply package.json:30]: Yes, atleast for the sake of logging, from my understanding. Not sure what else is effected by the `NODE_ENV=production`, if anything.

**Review by RasmusAlbrektsen — APPROVED** [review]

**Pull request merged**